staff/alicia-alcaide/spotify-template/index.html

```diff
@@ -0,0 +1,180 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Spotify</title>
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css?family=Montserrat" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css" />
+</head>
+<body>
+    <div class="main-container">
+        <header class="main-container__sidebar">
+            <nav class="nav">
+                <div class="logo">
+                    <i class="fab fa-spotify logo__icon"></i>
+                    <h1 class="logo__name">Spotify</h1>
+                </div>
+
+                <ul class="nav-items">
+                    <li class="item">
+                        <a href="#" class="item__link item__link--selected">
+                            <i class="fas fa-home link__icon link__icon--small"></i>
+                            <p class="link__name link__name--see-in-mobile">Inicio</p>
+                        </a>
+                    </li>
+                    <li class="item">
+                        <a href="#" class="item__link">
+                            <i class="fas fa-search link__icon link__icon--small"></i>
+                            <p class="link__name link__name--see-in-mobile">Buscar</p>
+                        </a>
+                    </li>
+                    <li class="item">
+                        <a href="#" class="item__link">
+                            <i class="fas fa-grip-lines-vertical  link__icon link__icon--small"></i>
+                            <p class="link__name link__name--see-in-mobile">Tu Bliblioteca</p>
+                        </a>
+                    </li>
+                    <li class="item">
+                        <a href="#" class="item__link">
+                            <i class="fas fa-arrow-alt-circle-down link__icon link__icon--small"></i>
+                            <p class="link__name link__name--see-in-mobile">Instalar app</p>
+                        </a>
+                    </li>
+                </ul>                           
+                
+                <div class="user-info item">
+                    <a href="#" class="user-info__link item__link">
+                        <i class="fas fa-user user-info__icon link__icon--small"></i>
+                        <p class="user-info__name link__name link__name--see-in-mobile">usuario</p>        
+                    </a>
+                </div>
+
+            </nav>
+        </header>
+
+        <main class="main-container__view">
+            <nav class="view__nav">
```

Comment:
@alicia-alcaide to keep BEM standard this would be, if your start with `main-container__view`, you should:
`main-container__view-nav`, `main-container__view-nav-items`, `main-container__view-nav-items-item`, `main-container__view-nav-items-item-link`.